cms/djangoapps/maintenance/views.py

```diff
@@ -0,0 +1,231 @@
+"""
+Views for the maintenance app.
+"""
+import logging
+from django.db import transaction
+from django.core.validators import ValidationError
+from django.core.urlresolvers import reverse_lazy
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext_lazy as _
+from django.views.generic import View
+
+from edxmako.shortcuts import render_to_response
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from contentstore.management.commands.utils import get_course_versions
+from util.views import require_global_staff
+
+
+log = logging.getLogger(__name__)
+
+# This dict maintains all the views that will be used Maintenance app.
+MAINTENANCE_VIEWS = {
+    "force_publish_course": {
+        "url": reverse_lazy("maintenance:force_publish_course"),
+        "name": _("Force Publish Course"),
+        "slug": "force_publish_course",
+        "description": _("Force publish course."),
+    },
+}
+
+
+COURSE_KEY_ERROR_MESSAGES = {
+    'empty_course_key': _('Please provide course id.'),
+    'invalid_course_key': _('Invalid course key.'),
+    'course_key_not_found': _('No matching course found.')
+}
+
+
+def get_maintenace_urls():
+    """
+    Returns all URLs for maintenance app.
+    """
+    url_list = []
+    for key, val in MAINTENANCE_VIEWS.items():  # pylint: disable=unused-variable
```

Comment:
1. instead of using `val`, use a variable name like `view_metadata`
2. I agree with @awaisdar001 , a list comprehension should be nicer here
3. if you're just using a value, do `MAINTENANCE_VIEWS.values()` or `MAINTENANCE_VIEWS.itervalues()`